_maintenance_update_graph_data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+outFile="$(dirname $0)/js/cert-timeline.tsv"
+
+# Download to a temp file, then move over-top the original, so that the update is
+# atomic from the perspective of the webserver
+curl --silent https://ct.tacticalsecret.com/cert-timeline.tsv >"${outFile}.bak"
+mv "${outFile}.bak" "${outFile}"
```

Comment:
Sure; I debated even having this script, though -  I figured Ops should pull this out, customize it, and delete it from the repo to avoid the chances of someone convincing us to merge a change with badness in it.

Still, updated in 7bcccd4.